src/backend/resourcemanager/utils/network_utils.c

```diff
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <time.h>
```

Comment:
You might add the following header file to suppress the warning about missing definition of on_proc_exit().

#include "storage/ipc.h"